package.json

```diff
@@ -20,7 +20,8 @@
     "@changesets/changelog-github": "^0.5.0",
     "@changesets/cli": "^2.27.1",
     "prettier": "^3.1.1",
-    "turbo": "^2.0.3"
+    "turbo": "^2.0.3",
+    "typedoc": "^0.28.1"
```

Comment:
I don't think we need to add this yet until we're actually using it in CI. We can do that separately.